bedrock/firefox/redirects.py

```diff
@@ -317,6 +317,86 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^firefox/upgrade', 'firefox.new'),
     redirect(r'^firefox/ie', 'firefox.new'),
 
+    # must go above the bug 1255882 stuff below
+    redirect('^projects/xul/joy-of-xul\.html$',
+             'https://developer.mozilla.org/docs/Mozilla/Tech/XUL/The_Joy_of_XUL'),
+    redirect('^projects/xul/xre(old)?\.html$',
+             'https://developer.mozilla.org/docs/Archive/Mozilla/XULRunner'),
+    redirect('^projects/xslt/js-interface\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Web/XSLT/Using_the_Mozilla_JavaScript_interface_to_XSL_Transformations'),
+    redirect('^projects/xslt/faq\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Web/API/XSLTProcessor/XSL_Transformations_in_Mozilla_FAQ'),
+    redirect('^projects/xslt/standalone\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Archive/Mozilla/Building_TransforMiiX_standalone'),
+    redirect('^projects/plugins/first-install-problem\.html$',
+             'https://developer.mozilla.org/Add-ons/Plugins/The_First_Install_Problem'),
+    redirect('^projects/plugins/install-scheme\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Installing_plugins_to_Gecko_embedding_browsers_on_Windows'),
+    redirect('^projects/plugins/npruntime-sample-in-visual-studio\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Compiling_The_npruntime_Sample_Plugin_in_Visual_Studio'),
+    redirect('^projects/plugins/npruntime\.html$',
+             'https://developer.mozilla.org/docs/Plugins/Guide/Scripting_plugins'),
+    redirect('^projects/plugins/plugin-host-control\.html$',
+             'https://developer.mozilla.org/docs/'
+             'Archive/Mozilla/ActiveX_Control_for_Hosting_Netscape_Plug-ins_in_IE'),
+    redirect('^projects/plugins/xembed-plugin-extension\.html$',
+             'https://developer.mozilla.org/Add-ons/Plugins/XEmbed_Extension_for_Mozilla_Plugins'),
+    redirect('^projects/netlib/http/http-debugging\.html$',
+             'https://developer.mozilla.org/docs/Mozilla/Debugging/HTTP_logging'),
+    redirect('^projects/netlib/integrated-auth\.html$',
+             'https://developer.mozilla.org/docs/Mozilla/Integrated_authentication'),
+    redirect('^projects/netlib/Link_Prefetching_FAQ\.html$',
+             'https://developer.mozilla.org/docs/Web/HTTP/Link_prefetching_FAQ'),
+    redirect(r'^projects/embedding/GRE\.html', 'http://developer.mozilla.org/en/GRE'),
+    redirect(r'^projects/embedding/windowAPIs\.html',
+             'http://developer.mozilla.org/en/Embedded_Dialog_API'),
```

Comment:
Maybe even replace `/en/` with `/docs/` to help MDN de-pollute it's namespace in the future?